manifest.yaml

```diff
@@ -1,10 +1,13 @@
 version: "2"
 
-name: Airdrop Template Snap-in
-description: A template for creating a new Airdrop snap-in
+# TODO: Replace with the name of your Airdrop connector
```

Comment:
I see you removed the template manifest.yaml and replaced it with this one, why? The point of the template one was that once you clone the template, some info is already filled in (like the name of the connector, based on the name of the repo).

Also make sure to update the Github actions file to reflect this change, there we go through all the template files and update and move them, so if you merge the template like this, without updating the script, stuff will break.